### DISCUSSION TITLE
Open source for "other collaborative projects"

### DISCUSSION THREAD

**Discussion by gruetter** [InnerSourceCommons/InnerSourceLearningPath#500] [General]
It is mentioned in this [this line](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/main/introduction/08-is-innersource-right.md) in document _Is InnerSource right for my project?_

> InnerSource projects make sense when the project is specific to the company or gives the company a strategic business advantage. Other collaborative projects should be run as open source to increase the contribution pool and impact.

I would argue that Open Source should only be done with the very same precondition: a business strategy  behind it. In that sense, I'd say InnerSource and Open Source are no different. For that reason, I think a clarification for this sentence would be helpful. WDYT?

**Comment by rrrutledge** [comment]
Sure - will take a look.

**Reply by rrrutledge** [reply]
Thanks for this feedback @gruetter ❗   I opened [Clarify when to InnerSource vs. open source](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/501) for this.  Have a look!
